src/main/java/io/debezium/connector/jdbc/dialect/db2/ZonedTimestampType.java

```diff
@@ -23,34 +18,28 @@
  *
  * @author Chris Cranford
  */
-public class ZonedTimestampType extends AbstractTimestampType {
+public class ZonedTimestampType extends io.debezium.connector.jdbc.type.debezium.ZonedTimestampType {
```

Comment:
I must admit, I'm not a fan of extending a class with the same name; should we think about prefixing the derived class with a database prefix of sorts?

Reply:
Or we can prefix the base one with "Debezium" as done for "Connect" one. WDYT? I am open for both.

Reply:
> Or we can prefix the base one with "Debezium" as done for "Connect" one.

Ya lets go with that approach, its more consistent with what we have already with prefixes.